docs/en/core-services/Caching.md

```diff
@@ -69,17 +69,18 @@ Alluxio storage is configured through Alluxio's configuration in `alluxio-site.p
 [configuration settings]({{ '/en/operation/Configuration.html' | relativize_url }}) for detailed
 information.
 
-A common modification to the default is to set the ramdisk size explicitly. For example, to set the
-ramdisk size to be 16GB on each worker:
+A common modification to the default is to set the ramdisk size explicitly.
+For example, to set the ramdisk size to be 16GB on each worker:
 
 ```properties
-alluxio.worker.ramdisk.size=16GB
+alluxio.worker.memory.size=16GB
```

Comment:
I think `ramdisk.size` is still the main property and `memory.size` is an alias? Also I see more places in the code using `PropertyKey.WORKER_RAMDISK_SIZE` than the latter. Do you mean `alluxio.worker.memory.size` is now replacing `alluxio.worker.ramdisk.size` now?

Reply:
oh you're right. That's my bad. I'll change it back